t/t3100-flux-in-flux.t

```diff
@@ -39,9 +39,9 @@ test_expect_success "flux --parent works in subinstance" '
 '
 
 test_expect_success "flux --parent --parent works in subinstance" '
-	id=$(flux submit \
-		flux start ${ARGS} \
-		flux start ${ARGS} flux --parent --parent kvs put test=ok) &&
+	id=$(flux batch -n1 --wrap \
+		flux run ${ARGS} flux start \
```

Comment:
One other comment sorry! Are `${ARGS}` meant for `flux start` and not `flux run` here?

Reply:
Oh duh.  Fix coming.